src/rest/nftmarket/module.ts

```diff
@@ -1,2 +1,38 @@
-// Todo: after swagger.yml change, implement and remove next line.
-export const DUMMY = 'dummy';
+import { QueryApi } from '../../openapi';
+import cosmosclient from '@cosmos-client/core';
+
+export function params(sdk: cosmosclient.CosmosSDK,) {
+  return new QueryApi(undefined, sdk.url).nftmarketParams();
+}
+
+export function nftListing(sdk: cosmosclient.CosmosSDK, classId: string, nftId: string) {
+  return new QueryApi(undefined, sdk.url).nftListing(classId, nftId);
+}
+
+export function listedNfts(sdk: cosmosclient.CosmosSDK) {
+  return new QueryApi(undefined, sdk.url).listedNfts()
+}
+
+export function listedClasses(sdk: cosmosclient.CosmosSDK, nftLimit?: number) {
+  return new QueryApi(undefined, sdk.url).listedClasses(nftLimit);
+}
+
+export function ListedClass(sdk: cosmosclient.CosmosSDK, classId: string, nftLimit: number) {
```

Comment:
Is there any reason why it is upper camel?

Reply:
Thank you for the notice.
I fixed it.